### PR TITLE
Copy stable validation policy to istiod-remote

### PR DESCRIPTION
**Pull request by whitneygriffith**
**Please provide a description of this PR:**

Part of [Release Channels: 173](https://github.com/istio/enhancements/issues/173)

### CODE REVIEW

File: Makefile.core.mk [hunk 1]
```diff
@@ -362,6 +362,7 @@ copy-templates:
 	cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates
 	cp manifests/charts/istio-control/istio-discovery/templates/_helpers.tpl manifests/charts/istiod-remote/templates
 	sed -e '1 i {{- if .Values.global.configCluster }}' -e '$$ a {{- end }}' manifests/charts/base/crds/crd-all.gen.yaml > manifests/charts/istiod-remote/templates/crd-all.gen.yaml
+	sed -e '1 i {{- if .Values.global.configCluster }}' -e '$$ a {{- end }}' manifests/charts/istio-control/istio-discovery/templates/validatingadmissionpolicy.yaml > manifests/charts/istiod-remote/templates/validatingadmissionpolicy.yaml
```
Comment (keithmattix) [inline Makefile.core.mk:365]: Do we want to include the policy in the base chart as well? The difference between the two is that the base chart is for the default revision

Comment (whitneygriffith) [reply Makefile.core.mk:365]: Oh, I didn't realize they were different. Yes, I will include both.

**Review by jaellio — COMMENTED** [review]
Are there stable configuration tests for each chart type? Do we verify no non-stable resources are present for each applied chart type?

**Comment by whitneygriffith** [comment]
> Are there stable configuration tests for each chart type? Do we verify no non-stable resources are present for each applied chart type?

I can't easily think of a way to automatically test that the expected manifests are in each chart type. I am open to suggestions and we can create an issue to track it.

**Review by MorrisLaw — APPROVED** [review]
lgtm

**Comment by whitneygriffith** [comment]
PTAL @ericvn @zirain @stevenctl

**Review by zirain — APPROVED** [review]

**Review by zirain — CHANGES_REQUESTED** [review]
For cherry-pick, please add release-notes.

**Review by keithmattix — APPROVED** [review]

**Review by stevenctl — APPROVED** [review]

**Comment by whitneygriffith** [comment]
> For cherry-pick, please add release-notes.

This is a bug fix that relates to release channels. There is already a [release notes](https://github.com/istio/istio/blob/master/releasenotes/notes/release-channels.yaml ) for release channels. Do I still need to add another?

**Review by zirain — APPROVED** [review]

**Pull request merged**